### ISSUE TITLE
themeReactable()

### ISSUE THREAD

**Issue by michaelheery** [glin/reactable#135]
Hi

i've been following the online tutorial to try and learn to use this package.  It's really great.  But when i get to the customizing theme part i try and recreate what is in the lesson but when I enter theme = reactableTheme() the reactableTheme function doesn't appear to have been installed as part of installing the package.  Does this still work?

Thanks

Michael

**Comment by glin** [comment]
Hi, `reactableTheme()` should be working, and I can think of a couple things to check:
1. Do you have a recent version of reactable installed? Check that `packageVersion("reactable")` shows 0.2.0 or later, as that was when theming was added.
2. Did you run `library(reactable)` to load the package first? If not, you can also use `reactable::reactableTheme()`

**Comment by michaelheery** [comment]
Greg

Thanks for getting in touch about this query.  I have reinstalled the package and it is now working!

I’m still very much learning my way in programming and your package is the first I’ve used moving beyond the Tidyverse universe.  I was always a little intimidated – especially where it starts to apply html which I don’t understand yet - but using the online guide I have been able to make good progress on it and really liking it!

Michael
